React/Views/RCTConvert+MapKit.m

```diff
@@ -66,4 +66,42 @@ + (RCTPointAnnotation *)RCTPointAnnotation:(id)json
 
 RCT_ARRAY_CONVERTER(RCTPointAnnotation)
 
++ (RCTPolyline *)RCTPolyline:(id)json
+{
+
+  json = [self NSDictionary:json];
+  NSArray* locations = (NSArray*)json[@"locations"];
```

Comment:
Please use this instead, as it does runtime type checking in dev mode:

   NSArray<NSDictionary *>* locations = [self NSDictionaryArray:json[@"locations"]];